VMEncryption/main/DiskUtil.py

```diff
@@ -735,6 +735,8 @@ def query_dev_sdx_path_by_uuid(self, uuid):
     def query_dev_id_path_by_sdx_path(self, sdx_path):
         """
         return /dev/disk/by-id that maps to the sdx_path, otherwise return the original path
+        Update: now we have realised that by-id is not a good way to refer to devices (they can change on reallocations or resizes).
+        Try not to use this- use get_stable_path_from_sdx instead
```

Comment:
are there any other places in the code calling this still?

Reply:
There is this one other place in OS encryption and it doesn't affect the azure_crypt_mount file.
So, I decided to not touch it (yet).

Reply:
OK